validations/src/mainnet_validations.rs

```diff
@@ -30,6 +31,10 @@ pub fn in_emergency_period(
     }
 }
 
+pub fn after_first_hard_fork(epoch: Epoch) -> bool {
+    epoch > FIRST_HARD_FORK
```

Comment:
What's "after"?

```suggestion
    epoch >= FIRST_HARD_FORK
```

Reply:
It was a question, I don't know if `>=` is more correct than `>`

Reply:
I am totally against triggering a semantic discussion on this haha